schema/backlog/-1.json

```diff
@@ -36,7 +36,7 @@
       "type": "string",
       "minLength": 0,
       "maxLength": 8,
-      "pattern": "^[ A-Za-z0-9]{0,8}$",
```

Comment:
The backlog directory is meant to be unchanged from what was generated by the script, so you'll want to leave out these changes.

Reply:
Oh i see that matt said to update the backlog.  Nevermind then

Reply:
Yeah, I changed my mind from the original proclamation made early on that backlog shouldn't be changed. It makes sense to update those as we go rather that remembering to change them if they get moved out of backlog. We definitely won't be re-running the script that created all those backlog files, though.